cdap-system-app-unit-test/src/test/java/io/cdap/cdap/test/SystemAppTestBaseTest.java

```diff
@@ -107,7 +107,9 @@ public void testSystemServiceInUserNamespaceFails() {
     }
   }
 
+  // TODO: CDAP-19775
   @Test
+  @Ignore
```

Comment:
How this test relates to CDAP-19775? It's deployment, it should not rely on versions. Please don't blank ignore all tests otherwise we end-up non-working branch

Reply:
`serviceManager.startAndWaitForGoodRun(ProgramRunStatus.RUNNING, 2, TimeUnit.MINUTES);` is related to program's version endpoints

Reply:
It's a test harness method that need to be updated to new API.
We should clearly separate test code that is calling specific api and this should continue to work after the change vs test harness code.